Example/Tests/UnsplashKitSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Quick
+import Nimble
+
+@testable import UnsplashKit
+
+extension SourceRequestFactory: Equatable {}
+public func ==(lhs: SourceRequestFactory, rhs: SourceRequestFactory) -> Bool {
+    return lhs === rhs
+}
+
+
+class UnsplashKitSpec: QuickSpec {
+    override func spec() {
+        
+        let requestFactory = SourceRequestFactory()
+                
+        describe("-init") {
```

Comment:
I don't think so since what the test coverage measures is how many times the code execution goes through that part of code. You construct this kit for testing so you're going through the constructor every time. @sergigracia